features/step_definitions/signup_steps.rb

```diff
@@ -63,11 +63,17 @@
   click_on 'Sign up'
 end
 
-When /^I fill in the invalid signup fields$/ do
+When /^I fill in the invalid signup fields( in a non-suspicious way)?$/ do |non_suspicious|
+  step %(15 seconds pass) if non_suspicious
   step %(I fill in "Email" with "invalid email")
   step %(I press "Sign up")
 end
 
+When /^I suspiciously fill in the signup fields as "([^"]*)"$/ do |name|
```

Comment:
Use cucumber expressions whenever possible:
```suggestion
When "I suspiciously fill in the signup fields as {string}" do |name|
```

Reply:
done